python/ray/dashboard/modules/job/job_head.py

```diff
@@ -317,7 +313,7 @@ async def upload_package(self, req: Request):
         try:
             data = await req.read()
             await get_or_create_event_loop().run_in_executor(
-                self._upload_package_thread_pool_executor,
```

Comment:
Is it critical enough to deserve its own thread pool?

Reply:
I don't think so because it's not latency critical either.